java/src/main/java/ai/rapids/cudf/MixedJoinSize.java

```diff
@@ -19,25 +19,22 @@
 /** This class tracks size information associated with a mixed table join. */
 public final class MixedJoinSize implements AutoCloseable {
   private final long outputRowCount;
-  // This is in flux, avoid exposing publicly until the dust settles.
-  private ColumnVector matches;
 
-  MixedJoinSize(long outputRowCount, ColumnVector matches) {
+  MixedJoinSize(long outputRowCount) {
     this.outputRowCount = outputRowCount;
-    this.matches = matches;
   }
 
   /** Return the number of output rows that would be generated from the mixed join */
   public long getOutputRowCount() {
     return outputRowCount;
   }
 
-  ColumnVector getMatches() {
-    return matches;
-  }
-
+  /**
+   * Close this resource. This implementation does nothing as there are no resources to release,
+   * but implementing AutoCloseable allows this class to be used in try-with-resources blocks.
+   */
```

Comment:
Since there is no resource to close, it should not be used in the `try` block. Any error/exception thrown out is still being handled properly. So we should not make this class `AutoClosable` anymore, to reduce mis-use.